test/propagatortest.jl

```diff
@@ -93,3 +94,35 @@ next_state_actual = expm(-im*QuDynamics.lindblad_op(sigmax, [lowerop(2)])*0.1)*v
 @test_approx_eq coeffs(vec(next_state_qexpokit_lmeq[1][2])) coeffs(next_state_actual)
 @test_approx_eq coeffs(vec(next_state_qexpokit_dm[1][2])) coeffs(next_state_actual)
 @test_approx_eq coeffs(vec(next_state_qexpokit_lmeq[1][2])) coeffs(next_state_actual)
+
+
+# Monte Carlo Wave Function Method
+Base.complex{B<:QuBase.OrthonormalBasis}(qarr::QuBase.AbstractQuArray{B}) = QuBase.similar_type(qarr)(complex(coeffs(qarr)), bases(qarr))
+rho = init_state_dm
+rhos = Array(typeof(rho), length(tlist)-1)
+ps = Array(typeof(rho), length(tlist)-1)
+qumcwfen = QuMCWFEnsemble(complex(init_state), 1000)
+
+for i=1:length(tlist)-1
+    rhos[i] = complex(zeros(rho))
+    ps[i] = complex(zeros(rho))
+end
+
+for psi0 in qumcwfen
+    i = 1
+    mcwf = QuMCWF()
+    for (t,psi) in QuPropagator(hamiltonian, c_ops, psi0, tlist, mcwf)
+        rhos[i] = rhos[i] + (psi*psi')/length(qumcwfen)/norm(psi)^2
+        i = i + 1
+    end
+end
+
+prop_expm = QuPropagator(hamiltonian, c_ops, rho, tlist, QuExpmV())
+j = 1
+for (t, psi) in prop_expm
```

Comment:
It might be better to rename `psi` to `rhot` or such, since this propagator actually evolves the density matrix and not a state vector.